docs/en/advanced-concepts/iwd_xcode7.md

```diff
@@ -0,0 +1,16 @@
+## Setting up instruments without delay for xcode 7 and iOS >= 9.0
+
+For iOS >= 9.0 instruments without delay does not work by passing binaries through
```

Comment:
Can you add the acronym here (i.e., `instruments without delay (iwd)`)? Judging from the project meeting today, it is not obvious that the two are related, though it may be more so in this context. I'd still like it to be explicit.